cluster/index.js

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env node
+/**
+ * Copyright 2016 Lyteworx LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+
+var cluster = require('cluster');
+var events = require('../red/runtime/events');
+var clusterRED = {};
+var cluster_events = require("events");
+var status_timeout = null;
+var last_worker = 0;
+var clusterphone = require("clusterphone");
+var _ = require('lodash');
+
+clusterRED._stop = false;
+clusterRED.cluster = cluster;
+clusterRED.clusterphone = clusterphone;
+clusterRED.events = new cluster_events.EventEmitter();
+clusterRED.events.setMaxListeners(100);
+clusterRED.debounce = {};
+clusterRED.contexts = {};
+clusterRED.clusterEnabled = false;
+process.bingo = false;
+
+clusterRED.getBingo = function(cb){
+    process.bingo = false;
+    for(var _id in cluster.workers){
+        var _w = cluster.workers[_id];
+        if(_w.process.bingo){
+            process.bingo = _w.process.pid;
+            break;
+        }
+        if(!process.bingo){
+            var _stuckee = clusterRED.getWorker();
+            process.bingo = _stuckee.process.pid;
+            _stuckee.process.bingo = true;
+        }    
+    }
+    return process.bingo;      
+}
+
+if(cluster.isWorker){
+    process.consolebackup = console.log;
+    console.log = function(){}

```

Comment:
One of the discussion points is what to do with duplicate messages from each worker, like the startup console messages.